config/webpack/entries/main.js

```diff
@@ -8,7 +8,7 @@
 import '@/public/dist/uib-templates';
 import angular from 'angular';
 
-if (process.env.NODE_ENV === 'production') {
+if (process.env.NODE_ENV !== 'development') {
   require('@/public/dist/templates');
```

Comment:
There are three environments, `development`, `test`, and `production` - I changed it here so we include the built templates for the test environment too.

The `npm run test:client` command was already building the templates (by running `gulp angular-templatecache`) but wasn't loading them. Hence all the 404 errors whilst running the tests - it used to mock specific templates that were loaded for specific tests, but that didn't catch all the cases where the templates were loaded.

Reply:
Aside; I wish we could slowly move away from `test` env to simplify things. Thanks for sorting out template loading!

Reply:
It's part of jest, so even if we wanted to (I like having the extra env), it would involve fighting jest:

https://github.com/facebook/jest/blob/9419034d6b575fe2c157453fe8b7d2000be66aad/packages/jest-cli/bin/jest.js#L12-L14

Reply:
Oh, interesting! I meant specifically that in our assets/config/Gulp setup we have the 3 envs separately — `test` being basically just copy of `development`: https://github.com/Trustroots/trustroots/blob/6c072ceab0be79b95f893600dc6bb8f1c50a7f53/config/assets/test.js

Reply:
Currently we have configurations that apply to:
- `development` only
- `production` only
- `development` or `test`

There are `development` only things that need to be off in `test`, so only have `production` only (and the inverse) wouldn't be sufficient. 

Perhaps we could just have `development` only (and the inverse).

Although once we get rid of karma/mocha _and_ gulp then becomes less relevant as the jest tests aren't put through webpack.

I'd suggest we just keep it in mind that at some point we could try and switch to only two varieties of env.